sdk/src/main/java/io/honeycomb/opentelemetry/HoneycombSdk.java

```diff
@@ -156,6 +157,51 @@ public Builder addSpanProcessor(SpanProcessor spanProcessor) {
             return this;
         }
 
+        /**
+         * Add a string attribute as a resource attribute.
+         *
+         * @return AttributesBuilder
+         */
+        public AttributesBuilder addResourceAttribute(String key, String value) {
```

Comment:
I think all of these javadocs need `@params`, otherwise doc generation fails with errors. We can probably configure that somehow, but that's the current default behavior.

Reply:
@vreynolds cool. Added `@param` comments.